packages/babel-traverse/src/path/inference/inferers.js

```diff
@@ -137,7 +137,13 @@ function Func() {
   return t.genericTypeAnnotation(t.identifier("Function"));
 }
 
-export { Func as Function, Func as Class };
+export {
+  Func as FunctionExpression,
```

Comment:
Would this be breaking if we remove `Func as Function, Func as Class`? (although maybe it's not used at all)